dds/DCPS/NetworkConfigUpdater.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
+#define OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
+
+#ifdef ACE_HAS_IOS 
+#define OPENDDS_NETWORK_CONFIG_UPDATER
+
+#include "ace/os_include/os_ifaddrs.h"
+#include "NetworkConfigMonitor.h"
+#include "dcps_export.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+class OpenDDS_Dcps_Export NetworkConfigUpdater : public NetworkConfigMonitor {
```

Comment:
I would expect this to follow the pattern of LinuxNetworkConfigMonitor and be called IOSNetworkConfigMonitor or something similar, unless I'm missing something?